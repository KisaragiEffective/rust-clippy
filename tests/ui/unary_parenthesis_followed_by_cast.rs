#![allow(unused)]
#![allow(clippy::unnecessary_cast)]
#![warn(clippy::unary_parenthesis_followed_by_cast)]

fn hello(arg_1: f64) {}

fn main() {
    // fire
    let x = 3.0f32;

    hello((x) as f64);
}
