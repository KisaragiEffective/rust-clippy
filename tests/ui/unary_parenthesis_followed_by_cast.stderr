error: unnecessary parenthesis
  --> $DIR/unary_parenthesis_followed_by_cast.rs:11:11
   |
LL |     hello((x) as f64);
   |           ^^^
   |
   = help: consider remove parenthesis
   = note: `-D clippy::unary-parenthesis-followed-by-cast` implied by `-D warnings`

error: aborting due to previous error

